rllib/algorithms/impala/impala_learner.py

```diff
@@ -60,8 +59,10 @@ def build(self) -> None:
                     framework=None,
                 )
             )
+            # As initial entropy coeff value, use the first timestep's (must be 0)
+            # value.
             self.curr_entropy_coeffs_per_module = defaultdict(
-                lambda: self._get_tensor_variable(self.hps.entropy_coeff)
+                lambda: self._get_tensor_variable(self.hps.entropy_coeff_schedule[0][1])
```

Comment:
This is a new concept:
For all schedules, lr, entropy, etc..
Do NOT use the non-schedule value anymore anywhere, not even for the initial value. Instead, the schedule is forced (via validate()) to have a 0-timestep entry, which defines the initial value. This is much more consistent and transparent for users: Either I use `lr` OR I use `lr_schedule`, but never both and there is no value-leakage between the two anymore. Proper validation code and descriptive error messages have been added.

Reply:
See my comment below. We check now in AlgorithmConfig (where it belongs).
I can add a comment explaining the [0][1], but I have to hard-code this! The first entry must start from ts=0 (to force more explicitness as to which value we start with). The definition of schedule configs is: List of tuples of the form `([ts], [lr to be reached at that ts])`.

Reply:
yes I agree. but you don't need to hardcode [0][1]. The check is already happening in validate() of algorithm config. So here we are assuming that that test has already been checked. Now I need readability and therefore:

```
_, init_entropy_coeff = next(iter(self.hps.entropy_coeff_schedule))
```

Reply:
You can classify this comment as nit :)